### DISCUSSION TITLE
Merge cmdline into statusline

### DISCUSSION THREAD

**Discussion by volgar1x** [nvim-lualine/lualine.nvim#634] [Ideas]
Hello, I think it would be so cool to be able to merge the cmdline with statusline, I imagine it would start showing over the filename

**Comment by shadmansaleh** [comment]
Not completely sure what you mean but https://github.com/neovim/neovim/pull/16251 might interest you. Both statusline & cmdlline are part of neovim ui. Lualine just customizes statusline . Merging cmdline & statusline is not something lualine can do .

**Reply by volgar1x** [reply]
Thanks for the link! I'll wait for the feature to be merged then :smile:
